content/en/cloud_cost_management/recommendations/_index.md

```diff
@@ -448,12 +448,12 @@ multifiltersearch:
 ---
 
 {{< callout url="#" btn_hidden="true" header="Join the Preview!" >}}
-Cloud Cost Recommendations is in Preview with support for AWS, Azure, and Google Cloud, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>
+Cloud Cost Recommendations is now GA with support for AWS, Azure, and Google Cloud, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>
```

Comment:
```suggestion
Cloud Cost Recommendations includes support for AWS, Azure, and Google Cloud, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>
```

Reply:
We should remove the callout entirely and move any relevant bits of this content to under ## Prerequisites